cattle/plugins/docker/compute.py

```diff
@@ -33,6 +34,7 @@ class DockerCompute(KindBasedMixin, BaseComputeDriver):
     def __init__(self):
         KindBasedMixin.__init__(self, kind='docker')
         BaseComputeDriver.__init__(self)
+        self.host_info = HostInfo()
```

Comment:
I'd prefer a stateless approach. I don't see any real need/huge benefit to making this an instance variable. I'd remove this line completely and change ```'physicalHostStats': self.host_info.collect_data()``` to ```'physicalHostStats':  HostInfo().collect_data()```.

For that matter, you could just have a function:
```
'physicalHostStats':  collect_host_data()
```
but that's primarily a stylistic choice and I won't push it.